scania-digital-design-system/sdds
# Added story for grid, spacing, new spacing option and more...

**Describe pull-request**  
_Describe what the pull-request is about_
- Added grid story
- Added new spacing option 72px css var
- Added spacing story
- Fixed some column issues in color story for mobile
- Added controls for the stories
- Expanded control information for stories
- Renamed typography story 


**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
fixes: #31 
fixes: #18 (bug)

**How to test**  
_Add description how to test if possible_
1. Run storybook
2. Check grid story
3. Try out different options, Enable addons in the settings of storybook, top left
4. Check spacing

**Additional context**  
_Add any other context about the pull-request here._
- Examples for grid might be a bit too complicated (?)
- Need to use the controls more :+1:
